### PR TITLE
Polkadot v0.9.39

### LINKED ISSUE #1435: Update upstream dependency to polkadot-v0.9.39
This is an issue to track the update of the upstream dependency.

### parachain dependency

- [x] Update local polkadot/substrate dependencies (helper script: `./scripts/bump-code-versions.sh`)
- [ ] Fix any compilation/test issues
- [ ] Optionally roll out the client/runtime update

### tee-worker dependency

- [x] Upstream PR: update [integritee-pallets](https://github.com/integritee-network/pallets) dependency
- [x] Upstream PR: update [integritee-node](https://github.com/integritee-network/integritee-node) dependency
- [x] Upstream PR: update [integritee-worker](https://github.com/integritee-network/worker) dependency
- [x] Ask integritee to update [frontier](https://github.com/integritee-network/frontier) and [substrate-api-client](https://github.com/scs/substrate-api-client) dependency
- [x] Merge upstream changes to `tee-worker/`

### PR DESCRIPTION
**Pull request by BillyWooo**
As title:
- updated upstream pallets
- updated upstream worker
- updated to polkadot-v0.9.39

ToDo:
upstream worker indirect call refactor

resolved: #1435

### CODE REVIEW

File: pallets/bridge-transfer/src/benchmarking.rs [hunk 1]
```diff
@@ -79,7 +79,7 @@ benchmarks! {
 	}:_(RawOrigin::Signed(sender),to_account,50u32.into(),resource_id)
 
 	set_maximum_issuance{
-		let origin = T::SetMaximumIssuanceOrigin::successful_origin();
+		let origin = T::SetMaximumIssuanceOrigin::try_successful_origin().expect("InitiateOrigin has no successful origin required for the benchmark");
```
Comment (wangminqi) [inline pallets/bridge-transfer/src/benchmarking.rs]: A little change of comment

File: pallets/drop3/src/benchmarking.rs [hunk 2]
```diff
@@ -97,7 +97,8 @@ fn create_default_proposal<T: Config>(caller: T::AccountId) -> (T::PoolId, Vec<u
 fn setup<T: Config>(should_start: bool) -> (T::AccountId, T::PoolId, Vec<u8>) {
 	let caller = create_default_caller::<T>();
 	let admin: T::AccountId = whitelisted_caller();
-	let origin = T::SetAdminOrigin::successful_origin();
+	let origin = T::SetAdminOrigin::try_successful_origin()
+		.expect("InitiateOrigin has no successful origin required for the benchmark");
```
Comment (wangminqi) [inline pallets/drop3/src/benchmarking.rs]: Same above

File: runtime/litentry/src/weights/pallet_democracy.rs [hunk 3]
```diff
@@ -262,4 +262,10 @@ impl<T: frame_system::Config> pallet_democracy::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+	fn set_external_metadata() -> Weight { todo!() }
+	fn clear_external_metadata() -> Weight { todo!() }
+	fn set_proposal_metadata() -> Weight { todo!() }
+	fn clear_proposal_metadata() -> Weight { todo!() }
+	fn set_referendum_metadata() -> Weight { todo!() }
+	fn clear_referendum_metadata() -> Weight { todo!() }
```
Comment (Kailai-Wang) [inline runtime/litentry/src/weights/pallet_democracy.rs]: We need to rerun the benchmarks

File: runtime/litmus/src/lib.rs [hunk 4]
```diff
@@ -346,8 +346,8 @@ impl pallet_timestamp::Config for Runtime {
 
 impl pallet_authorship::Config for Runtime {
 	type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Aura>;
-	type UncleGenerations = ConstU32<0>;
-	type FilterUncle = ();
+	// type UncleGenerations = ConstU32<0>;
+	// type FilterUncle = ();
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs]: We should remove the lines

File: tee-worker/.github/workflows/build_and_test.yml
Comment (Kailai-Wang) [inline tee-worker/.github/workflows/build_and_test.yml:1]: We should apply the changes to `ci.yml`, if not already

**Review by Kailai-Wang — APPROVED** [review]
Looks good in general, thanks. I didn't check every single line though.

Several small things are still missing. We are also a few commits away from the upstream latest

File: runtime/litentry/src/weights/pallet_democracy.rs (hunk 3 above)
Comment (wangminqi) [reply runtime/litentry/src/weights/pallet_democracy.rs]: done

File: tee-worker/.github/workflows/build_and_test.yml
Comment (BillyWooo) [reply tee-worker/.github/workflows/build_and_test.yml:1]: Yes. Updated.

**Pull request merged**